docs/reference/settings.md

```diff
@@ -764,6 +764,42 @@ formats described above.
 
 ---
 
+### [`fork-strategy`](#fork-strategy) {: #fork-strategy }
+
+The strategy to use when selecting multiple versions of a given package across Python
+versions and platforms.
+
+By default, uv will optimize for selecting the latest version of each package for each
+supported Python version (`requires-python`), while minimizing the number of selected
+versions across platforms.
+
+Under `fewest`, uv will minimize the number of
+selected versions for each package, preferring older versions that are compatible with a
+wider range of supported Python versions or platforms.
+
+**Default value**: `"fewest"`
```

Comment:
I'm seeing several references suggesting that `requires-python` is the default, but this says `fewest`.

Reply:
Thanks, that’s a mistake. Requires-Python is the default.

Reply:
Thanks! I'm happy for that to be the default!